### PR TITLE
99 add new organization

### PR DESCRIPTION
**Pull request by valentyntu**

### CODE REVIEW

File: client/src/modules/layouts/Contacts/ContactsPageTemplate.tsx [hunk 1]
```diff
@@ -14,22 +14,45 @@ interface Props {
   loadAggregatedContacts(): void;
 }
 
-class ContactsPage extends React.Component<Props, object> {
+class ContactsPage extends React.Component<Props, { open: boolean }> {
```
Comment (SamardakOleh) [inline client/src/modules/layouts/Contacts/ContactsPageTemplate.tsx]: name `openModal` is better name in this case

File: client/src/modules/layouts/Contacts/ContactsPageTemplate.tsx [hunk 2]
```diff
@@ -14,22 +14,45 @@ interface Props {
...
   public modelName: string;
   public noContactsMessage: string;
   public oneContactHeader: string;
   public moreThanOneContactsHeader: string;
   public columns: any[];
+  public signCreateNew: string;
+
+  public state = {
+    open: false,
+  };
```
Comment (SamardakOleh) [inline client/src/modules/layouts/Contacts/ContactsPageTemplate.tsx]: new line

File: client/src/modules/layouts/Contacts/Organizations/AddOrganization/AddOrganization.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+.dropdownClose {
+  composes: dropdown;
+}
+
+.menuItem {
+  display: flex;
+  align-items: center;
+  padding-left: 4px;
+  margin: 0 4px;
+  font: 400 14px/20px "Open Sans", sans-serif;
```
Comment (SamardakOleh) [inline client/src/modules/layouts/Contacts/Organizations/AddOrganization/AddOrganization.css]: 400 font is default

File: client/src/modules/layouts/Contacts/Organizations/AddOrganization/AddOrganization.tsx [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+          customField.model === 'Organization' && customField.isDefault && customField.name === 'Address');
+      const organization = {
+        custom: [
+          {
+            key: customFieldSetting ? customFieldSetting._id : '',
+            value: this.state.address,
+          },
+        ],
+        name: this.state.name,
+        owner: this.props.auth.userid,
```
Comment (SamardakOleh) [inline client/src/modules/layouts/Contacts/Organizations/AddOrganization/AddOrganization.tsx]: better to get this value from authentificated user in backend

File: api/organization/index.js [hunk 5]
```diff
@@ -56,7 +56,8 @@ router.post("/", (req, res) => {
     _id: new mongoose.Types.ObjectId(),
     name: req.body.name,
     domain: req.user.domain,
-    owner: req.user._id,
+    custom: req.body.custom,
+    owner: "owner" in req.body ? req.body.owner : req.user._id,
```
Comment (alexanderko) [inline api/organization/index.js]: use only logged in user as owner

**Pull request merged**